src/app/containers/Video/helpers/fixtures.js

```diff
@@ -0,0 +1,556 @@
+          subType: 'clip',
+          syndication: {
+            destinations: [],
+          },
+          synopses: {
+            short:
+              'They may be tiny, but us humans could learn a thing or two from ants.',
+          },
+          title: 'Five things ants can teach us about management',
+          versions: [
+            {
+              availableTerritories: {
+                nonUk: true,
+                uk: true,
+              },
+              availableUntil: null,
+              duration: 191,
+              types: ['Original'],
+              versionId: 'p01k6msp',
+              warnings: {
+                long: 'Contains strong language and adult humour.',
+                short: 'Contains strong language and adult humour.',
+              },
+            },
+          ],
+        },
+        type: 'aresMediaMetadata',
+      },
+      {
+        model: {
+          blocks: [
+            {
+              model: {
+                copyrightHolder: 'BBC',
+                height: 1080,
+                locator: 'ichef.test.bbci.co.uk/images/ic/$recipe/p01k6mtv.jpg',
+                originCode: null,
+                width: 1920,
+              },
+              type: 'rawImage',
+            },
+            {
+              model: {
+                blocks: [singleTextBlock('Ants')],
+              },
+              type: 'altText',
+            },
+          ],
+        },
+        type: 'image',
+      },
+    ],
+  },
+  type: 'aresMedia',
+};
+
+export const captionBlock = {
+  model: {
+    blocks: [
+      singleTextBlock(
+        'p01k6msm: Video, Clip, UK and non-UK, guidance, subtitles (about bees)',
+      ),
+    ],
+  },
+  type: 'caption',
+};
+
+export const videoPortraitBlock = {
+  model: {
+    blocks: [
+      {
+        blockId: 'urn:bbc:ares::clip:p01m7hmc',
+        type: 'aresMediaMetadata',
+        model: {
+          id: 'p01m7hmc',
+          subType: 'clip',
+          format: 'audio_video',
+          title: 'Test vertical video',
+          synopses: {
+            short: 'This is a really plain green test vertical video',
+          },
+          imageUrl: 'ichef.test.bbci.co.uk/images/ic/$recipe/p01m7hny.jpg',
+          imageCopyright: null,
+          embedding: true,
+          advertising: true,
+          caption: null,
+          versions: [
+            {
+              versionId: 'p01m7hmf',
+              types: ['Portrait'],
+              duration: 11,
+              warnings: {},
+              availableTerritories: {
+                uk: true,
+                nonUk: true,
+              },
+              availableUntil: null,
+            },
+          ],
+          image: null,
+          syndication: {
+            destinations: [],
+          },
+        },
+      },
+      {
+        type: 'image',
+        model: {
+          blocks: [
+            {
+              type: 'rawImage',
+              model: {
+                width: null,
```

Comment:
Do we not get width and height values here?

Reply:
No we don't. The origin of this image is not CPS Optimo. It is MPV. If you look at the locator, it does not have a value for height or width specified since when passed into the media player it will use the relevant height or width necessary for the device.